fe/src/components/layouts/Default/Default.tsx

```diff
@@ -1,19 +1,22 @@
-import { Box } from '@chakra-ui/react';
+import { Box, Flex } from '@chakra-ui/react';
 import React from 'react';
 import Header from '../../Header';
 import Footer from '../../Footer';
+import Sidebar from '../../Sidebar';
 
-export default function Default({ children } : { children: React.ReactNode }): JSX.Element {
+export default function Default({
+  children,
+}: {
+  children: React.ReactNode;
+}): JSX.Element {
   return (
     <>
-      <Box
-        position="relative"
-        minHeight="100%"
-      >
+      <Box position="relative" minHeight="100%">
         <Header />
-        <Box pb={14}>
-          {children}
-        </Box>
+        <Flex position="sticky" flexDirection="row" pb={14}>
+          <Sidebar />
+          <Flex w="100%">{children}</Flex>
```

Comment:
A small nitpick for now, but page children should not be displayed inside a Flex row element, a Box would do just fine.